system/api-server/src/app.controller.ts

```diff
@@ -10,3 +10,4 @@ export class AppController {
     return this.appService.getHello();
```

Comment:
Same here, boilerplate that should be removed

Reply:
Alright.